src/ui/widgets/messagebox.h

```diff
@@ -38,6 +40,11 @@ class MessageBox : public Widget
     return m_confirmed;
   }
```

Comment:
```suggestion
}
```

Reply:
Are you sure about that? Looks legit to me.